app/views/kaminari/_paginator.html.erb

```diff
@@ -0,0 +1,17 @@
+  <%= paginator.render do %>
+  <nav>
+      <ul class="pagination justify-content-center">
+        <%= first_page_tag unless current_page.first? %>
+        <%= prev_page_tag unless current_page.first? %>
+        <% each_page do |page| %>
+        <% if page.left_outer? || page.right_outer? || page.inside_window? %>
```

Comment:
This is one of the auto generated files, right?

The indentation looks a little off. Is it possible to fix it?

Reply:
Yes, it is. It looks to be centered to the content div, but let me check and play around with it.

Reply:
<img width="1218" alt="center" src="https://user-images.githubusercontent.com/8164007/33222160-50a6eeb6-d112-11e7-928d-0d5ad4f3c653.png">

So, I'm pretty sure it is horizontally centered. I did add some top margins so it's uniform to the bottom and not hugging the line below the item

Reply:
Sorry, I was referring to code indentation not the alignment.

Reply:
Gotcha, my mistake. Fixed with latest push